task/attest/src/main.rs

```diff
@@ -93,9 +92,8 @@ struct AttestServer {
 
 impl Default for AttestServer {
     fn default() -> Self {
```

Comment:
This isn't new in your change, but a Default impl that can only ever be called once seems sketchy.

Reply:
yeah, i also noticed and didn't love that; I wasn't sure if it was worth changing as part of this branch or not, but i may go ahead and do it here.